src/components/DocumentProducer/DocumentMetadataEdit/index.tsx

```diff
@@ -1,14 +1,19 @@
-import React, { useEffect, useMemo } from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
```

Comment:
@nickgraz is it ok to modify this file directly since it's an existing document producer component or should we clone it?

Reply:
Yea this is perfect